crates/pallets/starknet/src/lib.rs

```diff
@@ -157,6 +157,8 @@ pub mod pallet {
         type ProtocolVersion: Get<u8>;
         #[pallet::constant]
         type ProgramHash: Get<Felt252Wrapper>;
+        #[pallet::constant]
+        type WhitelistedClassHashes: Get<Vec<CasmClassHash>>;
```

Comment:
why do we've a pallet constant and a storage variable?

Reply:
should we also add a vector in the genesis config where the chain can set the class hashes on start?

Reply:
Hm yea we can probably do that, in this case I'll remove the constant then right ?